lib/rubycritic/cli/options.rb

```diff
@@ -19,6 +19,16 @@ def parse
             @root = path
           end
 
+          opts.on('-b', '--branch BASE_BRANCH,FEATURE_BRANCH', 'Set branches to compare') do |branches|
+            self.base_branch, self.feature_branch = branches.split(',').map { |branch| String(branch) }
+            self.mode = :compare_branches
+          end
+
+          opts.on('-t', '--threshold-score [THRESHOLD_SCORE]',
+                  'Set a threshold for score difference between two branches (works only with -b)') do |threshold_score|
+            self.threshold_score = Integer(threshold_score)
```

Comment:
It's like codecov's patch difference. If the patch contains too many new code smells we reject it.